src/DocScan/Request/Check/Report/SandboxDetail.php

```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
 
-namespace Yoti\Sandbox\DocScan\Request;
+namespace Yoti\Sandbox\DocScan\Request\Check\Report;
 
-class SandboxDetails implements \JsonSerializable
+class SandboxDetail implements \JsonSerializable
```

Comment:
Just looking at .NET and Java, and we have a bit of mix and match - this class is called `SandboxDetails` in those, but we still have `WithDetail` in the BreakdownResponseBuilder. I _think_ I remember @MrBurtyyy changing it from SandboxDetail -> SandboxDetails. Do you remember Alex?

Reply:
It was changed in Java recently to singular https://github.com/getyoti/yoti-java-sdk/blob/DEVELOPMENT/yoti-sdk-sandbox/src/main/java/com/yoti/api/client/sandbox/docs/request/check/report/SandboxBreakdown.java#L65

I idea being details is a list of "detail" - it also allows for `withDetails(details)` (list) to be added in future, although that could just be named `withDetailsList()`

Reply:
As discussed, let's stick with `SandboxDetail`